frontend/src/scenes/web-analytics/WebAnalyticsFilters.tsx

```diff
@@ -52,6 +51,7 @@ const CondensedWebAnalyticsFilterBar = ({ tabs }: { tabs: JSX.Element }): JSX.El
                     <ReloadAll iconOnly />
                     <DateFilter allowTimePrecision dateFrom={dateFrom} dateTo={dateTo} onChange={setDates} />
                     <WebAnalyticsCompareFilter />
+                    <FilterPresetsDropdown />
```

Comment:
I feel like this should be closer to the filters, or even inside the filters dropdown, to select a preset, with maybe a CTA to save the current filters as a preset... but this might be a stretch. 

But... it is just a feeling that filters + presets should live closer together.

I think the collapsible filters are better than what was there before, and maybe the floating component is not the way to go forward as things have more options... but 

It would be nice to run this through @PostHog/platform-ux or Cory as they might have some ideas on how other products solved it.